### PR TITLE
new vars for AWS credentials for Dynamodb

### PR DESCRIPTION
**Pull request by Baggerone**
[IDP-512](https://itse.youtrack.cloud/issue/IDP-512) provide separate credentials for DynamoDB access in simplesamlphp-module-sildisco

### CODE REVIEW

**Review by dalenewby — APPROVED** [review]

File: vars.tf [hunk 1]
```diff
@@ -48,6 +48,16 @@ variable "docker_tag" {
   default = "latest"
 }
 
+variable "dynamo_access_key_id" {
+  description = "AWS credential for DynamoDB permissions for sildisco to log user logins"
```
Comment (Schparky) [inline vars.tf:52]: Where is this key going to be created? Would it make sense to manage it in this workspace?

Comment (Baggerone) [reply vars.tf:52]: I hadn't thought that far ahead.  If you think that's the best solution, I'll try to include that.

Comment (Schparky) [reply vars.tf:52]: I can't think of a better place. That would save the manual step of copying a key from elsewhere into this variable.

File: main.tf [hunk 2]
```diff
@@ -181,6 +183,39 @@ module "ecs" {
...
+ * Create key for dynamo permissions
+ */
+resource "aws_iam_access_key" "user_login_logger" {
+  user    = aws_iam_user.user_login_logger.name
+}
+
+/*
+ * Allow user_login_logger user to write to Dynamodb
+ */
+resource "aws_iam_role_policy" "dynamodb-logger-policy" {
```
Comment (Schparky) [inline main.tf:203]:
```suggestion
resource "aws_iam_user_policy" "dynamodb-logger-policy" {
```

File: main.tf [hunk 3]
```diff
@@ -181,6 +183,39 @@ module "ecs" {
...
+   name = "dynamodb_user_login_logger_policy-${local.app_env}"
+   user = aws_iam_user.user_login_logger
```
Comment (Schparky) [inline main.tf:205]:
```suggestion
   user = aws_iam_user.user_login_logger.name
```

**Review by Schparky — APPROVED** [review]
Very good, just a couple small corrections.

**Pull request merged**